Apps/WebClient/src/ClientApp/src/components/public/sharing-invite/SharingInviteView.vue

```diff
@@ -0,0 +1,77 @@
+<script setup lang="ts">
+import { computed } from "vue";
+import { RouteLocationRaw, useRouter } from "vue-router";
+
+import HgButtonComponent from "@/components/common/HgButtonComponent.vue";
+import { useAuthStore } from "@/stores/auth";
+import { useUserStore } from "@/stores/user";
+
+interface Props {
+    inviteId: string;
+}
+const props = defineProps<Props>();
+
+const router = useRouter();
+const authStore = useAuthStore();
+const userStore = useUserStore();
+
+const oidcIsAuthenticated = computed(() => authStore.oidcIsAuthenticated);
+const invitePath = computed(() => `/sharing?invite=${props.inviteId}`);
+const loginRoute = computed<RouteLocationRaw>(() => ({
+    path: "/login",
+    query: { redirect: invitePath.value },
+}));
+const registrationRoute = computed<RouteLocationRaw>(() => ({
+    path: "/registration",
+    query: { redirect: invitePath.value },
```

Comment:
the current convention with router paths is to use the Path constants `Path.Login` and `Path.Registration`